src/lib.rs

```diff
@@ -67,6 +67,11 @@ pub use crate::error::{Error, Result};
 
 #[macro_use]
 mod cfg;
+
+#[macro_use]
+#[cfg_attr(docsrs, doc(all(feature = "tracing"), hyper_unstable_tracing))]
```

Comment:
Is this attribute required? Since it's not public, the module won't be rendered on docs.rs.